packages/shuttle/src/shuttle/db.ts

```diff
@@ -140,9 +144,27 @@ type MessagesTable = {
 export type MessageRow = Selectable<MessagesTable>;
 export type InsertableMessageRow = Insertable<MessagesTable>;
 
+type OnChainEventsTable = {
+  id: Generated<string>;
+  chainId: bigint;
+  createdAt: Generated<Date>;
+  updatedAt: Generated<Date>;
+  blockTimestamp: Date;
+  fid: Fid;
+  blockNumber: bigint;
+  logIndex: number;
+  type: OnChainEventType;
+  txHash: Uint8Array;
+  body: IdRegisterEventBody | SignerEventBody | StorageRentEventBody;
```

Comment:
```suggestion
  body: IdRegisterEventBody | SignerEventBody | SignerMigratedEventBody | StorageRentEventBody;
```

For completeness, since by default backfill does request this type of on-chain event from the hub (https://github.com/neynarxyz/hub-monorepo/pull/2/files#diff-8b92dfe3afde86f73f1479c2a6a5ad501444213839bf49202a0fdd161d8e6408R59)